README.md

```diff
@@ -193,6 +193,42 @@ dbt-invoke properties.delete <options>
   - `<options>` uses the same arguments as for creating/updating property
     files, except for `--threads`.
 
+
+### Applying default properties using `dbt_invoke_template.yml`
+
+You can pass custom default properties to be added to property files on update.
+To do so, add a `dbt_invoke_template.yml` at the root of your dbt project,
+using the following convention:
+
+
+```
+model:
+  meta:
+    owner: "@default"
+  columns:
+    foo: "bar"
```

Comment:
Can you help me understand more about the use case of adding custom default properties to columns. In this example, does the `foo: "bar"` property get added to every column?

Reply:
Exactly, here's an example to be a little clearer:

Create a dummy table 

<img width="1260" alt="Screenshot 2023-10-11 at 10 35 27" src="https://github.com/Dashlane/dbt-invoke/assets/115995643/f0eb7eea-d732-4512-8c14-6169035eaab5">

-----

Generate property file using dbt-invoke with a template

<img width="852" alt="Screenshot 2023-10-11 at 10 39 19" src="https://github.com/Dashlane/dbt-invoke/assets/115995643/74a32c47-84fa-4c75-b84d-0ea3b5b61d36">


-----

To be honest, I'm not planning on using this just yet, but saw the opportunity of adding this extra thing and thought maybe someone would find it handy. We could remove this from the PR if you feel like the value isn't 100% clear 😊

Reply:
Thanks for the example! I'm realizing that having default properties on columns was actually requested as part of https://github.com/Dashlane/dbt-invoke/issues/7#issue-975223699, so I think it's worth keeping.